### ISSUE TITLE
How to share binary content?

### ISSUE THREAD

**Issue by matthiasbeyer** [hfuunfug/unfug.org#31]
We do not have a solution yet: How to share the PDFs and the other binary content. I won't check them into git, that's for sure.

**Comment by matthiasbeyer** [comment]
@fkemmer opts for git-annex, as I do.

**Comment by TheNeikos** [comment]
You do realize that with `git-annex` you just put in the index of a file rather than it's contents, making it useless to have in this case?

**Comment by matthiasbeyer** [comment]
No. We would put the annex directory (the git repository) on github, but the content (the real files) onto unfug.org. This way, one is able to clone the repository from github and then `git annex get file.pdf` to get the real files.

(The CCC does it this way, for example. You can download the whole index of CCC videos from github and then pull the movies and stuff from media.ccc.de)

**Comment by TheNeikos** [comment]
Well then, I must have misread (or not enough) the pieces I saw about it. In that case I would say go for it.

**Comment by matthiasbeyer** [comment]
For future reference: https://git-annex.branchable.com/tips/setup_a_public_repository_on_a_web_site/

**Comment by kallies** [comment]
Something like Git LFS might also do the trick.
